sources/assets/Stride.Core.Assets/Stride.Core.Assets.csproj

```diff
@@ -29,6 +29,7 @@
   </ItemGroup>
   <ItemGroup>
     <ProjectReference Include="..\..\buildengine\Stride.Core.BuildEngine.Common\Stride.Core.BuildEngine.Common.csproj" />
+    <ProjectReference Include="..\..\core\Stride.Core\Stride.Core.csproj" />
```

Comment:
Curious, you remember why it was necessary?

Reply:
it used to be broken references? seems to work without now.